processing/src/main/java/org/apache/druid/query/groupby/having/GreaterThanHavingSpec.java

```diff
@@ -119,4 +123,36 @@ public String toString()
     sb.append('}');
     return sb.toString();
   }
+
+  @Override
+  public byte[] getCacheKey()
+  {
+    try {
+      final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
Would CacheKeyBuilder work for this (and similar method)?

Reply:
Yeah, I can use CacheKeyBuilder, wonder why I didn't earlier.